### PR TITLE
Skip verify video in home feed

### PR DESCRIPTION
**Pull request by YT-Advanced**
Currently, the Alt Thumbnails take lots of time for showing the thumbnails, this PRs will enhance the loading time in home feed

### CODE REVIEW

**Comment by YT-Advanced** [comment]
@inotia00 It is ready to merge right now

**Comment by inotia00** [comment]
Checking BrowseId has been removed for reasons mentioned above
I will merge the PR

**Comment by YT-Advanced** [comment]
@inotia00 Where is the reason, I cannot found it

**Comment by inotia00** [comment]
review_outdated
I removed the last commit with a force push, and it appears that comments left on the commit were also removed during this process.

**Comment by YT-Advanced** [comment]
@inotia00 My idea is skip Verifying Thumbnails in Home Feed. But Livestream Video will have blank thumbnails. Luckily, we can prevent it by checking if a thumbnails contains `_live.` or not. However, thumbnails in Suggestion Shelf is an exception, it don't have the parameter above and currently we don't have any way to check if a thumbnails is generated by Suggestion Shelf or not. That's why I check the HIDE_SUGGESTION_SHELF value.

Sorry because of my bad English :(

**Comment by inotia00** [comment]
> review_outdated

Even if the `Hide suggested shelf` patch is included, we don't need to use this statement

```
BrowseIdPatch.isHomeFeed() && SettingsEnum.HIDE_SUGGESTIONS_SHELF.getBoolean()
```

If `HIDE_SUGGESTIONS_SHELF` is enabled, the suggestions shelves will be hidden.
so thumbnails for suggestions shelves videos are not be loaded

**Comment by YT-Advanced** [comment]
> Even if the `Hide suggested shelf` patch is included, we don't need to use this statement
> 
> ```
> BrowseIdPatch.isHomeFeed() && SettingsEnum.HIDE_SUGGESTIONS_SHELF.getBoolean()
> ```

The function of the inner statement is to skip thumbnails verification. That's why it returns true

> If `HIDE_SUGGESTIONS_SHELF` is enabled, the suggestions shelves will be hidden.
> so thumbnails for suggestions shelves videos are not be loaded

Yeah. When `HIDE_SUGGESTIONS_SHELF` is enabled, and the user is in Home Feed, we don't need to send HEAD request for verifying the thumbnails


### I will explain more about my idea. When you turned off the thumbnails verifying, the thumbnails will blank if:
- Video is livestream
- Video is very old
- Video have very low view (about 0-20)
- Recent uploaded video

### In Home Feed:
- Livestream video can be checked by `.live_`.
But as I said above, livestream video in Suggestion Shelf can't be checked via this way. That's why I check the boolean of Hide Suggestion Shelf.
- 3 others ones that I mentioned above have very rare rate to be shown, so we can skip

**Comment by inotia00** [comment]
### In Home Feed:
- Livestream video can be checked by `.live_`.
But as I said above, livestream video in Suggestion Shelf can't be checked via this way. That's why I check the boolean of Hide Suggestion Shelf.
- 3 others ones that I mentioned above have very rare rate to be shown, so we can skip

I understand why you added this code, but it's not appropriate

`Hide suggestion shelf` is based on litho components, so we can only hide suggestion shelf
(suggestion shelf filter is enabled when [horizontalShelf](https://github.com/inotia00/revanced-integrations/blob/6046fc29ecc0fdf71d0685651414722fb7c6c1c8/app/src/main/java/app/revanced/integrations/patches/components/SuggestionsShelfFilter.java#L24) is detected)

Alternative thumbnail patch does not use litho components

In other words, If the user is in the home feed and the HIDE_SUGGESTIONS_SHELF is enabled, the thumbnail verification of all videos will be ignored (we cannot check whether the video belongs to the suggestion shelf because it is not using the litho component)

```
BrowseIdPatch.isHomeFeed() && SettingsEnum.HIDE_SUGGESTIONS_SHELF.getBoolean()
```

Therefore, this is not an appropriate method.

To check if the live stream video belongs to suggestion shelf, litho component filter is necessary

**Comment by inotia00** [comment]
For this reason, I will roll back this PR

**Comment by YT-Advanced** [comment]
> To check if the live stream video belongs to suggestion shelf, litho component filter is necessary

If so, do you have any others way to check ? Since if we filtering with litho, it is difficult for us to know exactly which thumbnails belong to which video

> For this reason, I will roll back this PR

I wonder what PRs that you mean in this case xD. Do you mean recent commit that you merged or roll back the force-push.
If you are meaning the commit, I think it is uneccessary
